pkgs/development/libraries/tiledb/default.nix

```
@@ -17,6 +17,7 @@
 , gtest
 , doxygen
 , fixDarwinDylibNames
+, useAVX2 ? false
```

```suggestion
, useAVX2 ? stdenv.hostPlatform.avx2Support
```

Should allow it to be automatically enabled if someone's building nixpkgs for a specific target processor.

---

@risicle Is this `hostplatform.avx2Support` setting then also disabled for all builders that produce the packages for nixkpgs? Otherwise it defeats the purpose of getting reproducible packages in the cache.

---

Yes. This is "microarchitecture level support" in nix and the default is x86-64-v1.